include/amqpcpp/libboostasio.h

```diff
@@ -468,24 +513,24 @@ class LibBoostAsioHandler : public virtual TcpHandler
     /**
      *  Method that is called when the heartbeat frequency is negotiated between the server and the client.
      *  @param  connection      The connection that suggested a heartbeat interval
-     *  @param  interval        The suggested interval from the server
-     *  @return uint16_t        The interval to use
+     *  @param  timeout         The suggested timeout from the server
+     *  @return uint16_t        The timeout to use
      */
-    virtual uint16_t onNegotiate(TcpConnection *connection, uint16_t interval) override
+    virtual uint16_t onNegotiate(TcpConnection *connection, uint16_t timeout) override
     {
         // skip if no heartbeats are needed
-        if (interval == 0) return 0;
+        if (timeout == 0) return 0;
 
         const auto fd = connection->fileno();
 
         auto iter = _watchers.find(fd);
         if (iter == _watchers.end()) return 0;
 
         // set the timer
-        iter->second->set_timer(connection, interval);
+        iter->second->set_timers(connection, timeout);
 
-        // we agree with the interval
-        return interval;
+        // we agree with the timeout
+        return timeout;
```

Comment:
This renaming is to achieve consistency with libev.h.

Although "interval" is the most used term in the library, I feel that calling "interval" wouldn't be intuitive to read "heartbeat interval = interval / 2". So I agree more with the "timeout" term chosen in libev.h